borg/archiver.py

```diff
@@ -17,6 +18,8 @@
 from . import __version__
 from .archive import Archive, ArchiveChecker, CHUNKER_PARAMS
 from .compress import Compressor, COMPR_BUFFER
+from .logger import create_logger, setup_logging
+logger = create_logger()
```

Comment:
logging module is a bitch. you must not create loggers and afterwards try to setup/configure them, that does not work.

so the rule is "configure first, then start creating loggers".